Source/Renderer/Context.js

```diff
@@ -508,6 +509,18 @@ define([
         },
 
         /**
+         * <code>true</code> if the WebGL context supports stencil buffers.
+         * Stencil buffers are not supported by all systems.
+         * @memberof Context.prototype
+         * @type {Boolean}
+         */
+        stencilBuffer : {
+            get : function() {
+                return this._stencilBits >= 8;
```

Comment:
When the context is created without `stencil : true`, `gl.getParameter(gl.STENCIL_BITS);` returns 0 so this test should be fine.